### ISSUE TITLE
Gamma setting issue

### ISSUE THREAD

**Issue by DarkGuardsman** [FoxBytes/SpaceTruckerBugs#54]
Gamma seems to effect the entire PC rather than just the game, Might just be an issue with alt-tabbing

**Comment by Maxwolf** [comment]
The problem here is that it is just to dark in the prototype. I realized this after I released it and fixed my monitors color profile and fixed this so the game is not as dark and gamma should not need to be changed to see where you are going now. The prototype is also pretty dark in general as more lights need to be placed down on maps as they are filled out.

The gamma setting changing entire desktop is not something I can actually mess with since we use the OGRE rendering engine and that is the tactic which that renderer uses to increase gamma. I could redo it with a shader instead to make things brighter but overall easier to make game correct brightness.

Should not be a problem in the next build!
